app/views/projects/merge_requests/show/_mr_accept.html.haml

```diff
@@ -40,12 +40,12 @@
   .automerge_widget.no_satellite.hide
     %p
       %span
-        %strong This repository does not have satellite. Ask an administrator to fix this issue
+        %strong This repository does not have a satellite. Please, ask an administrator to fix this issue!
 
   .automerge_widget.cannot_be_merged.hide
     %h4
-      This request can't be merged with GitLab.
-      You should do it manually with
+      This request cannot be merged with automatically.
```

Comment:
I don't think this line is correct. What about:
"This request cannot be merged automatically with GitLab."?